src/System.Diagnostics.StackTrace/src/System/Diagnostics/StackTraceSymbols.CoreCLR.cs

```diff
@@ -210,9 +210,15 @@ void IDisposable.Dispose()
                 return null;
             }
 
+            if (string.IsNullOrEmpty(path))
+            {
+                return null;
+            }
```

Comment:
Why this change?

Reply:
https://dev.azure.com/dnceng/public/_build/results?buildId=211548&view=logs&jobId=77593f2f-9c1d-5a27-f56c-e4d6cda477a7

Reply:
Should I just bang it away?

Reply:
> Should I just bang it away?

Yup.

Reply:
(When we eventually attribute File.Exists, the argument will be attributed as [NotNullWhen(true)], and then the ! won't be necessary.)